### DISCUSSION TITLE
Search for UNICODE string?

### DISCUSSION THREAD

**Discussion by LordRadai** [NationalSecurityAgency/ghidra#7096] [Q&A]
Hello, with this release I cannot seem to find a way to search for a Unicode string in the Search Memory widget... the old version let you choose what kind of string formatting you wanted to use, this one just defaults to ASCII. Am I missing something or was it removed?

**Comment by dragonmacher** [comment]
In the new search dialog you select 'String' in the first drop-down widget.  Then, in the Search Options panel (which  you may have to open), you can select UTF-8 or UTF-16 for the Encoding under String Options.

![image](https://github.com/user-attachments/assets/e6262fc8-6e59-4cf7-a57e-7977a3d74af1)
